components/31-molecules/breadcrumbs/breadcrumbs.functions.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+/* global define, module */
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(factory);
+  }
+  else {
+    if (typeof exports === 'object') {
+      module.exports = factory();
+    }
+    else {
+      root.Breadcrumbs = factory();
+    }
+  }
+}(this || window, function () {
+
+  return function (elem, options) {
+    let list = elem.querySelector('ol, ul');
+    let items = elem.querySelectorAll('li');
+    let self = this;
+
+    /**
+     * Add an element to the DOM that makes it possible to expand the breadcrumb.
+     *
+     * @param {Int} position
+     *   Index of the new element.
+     */
+    const insertExpandable = position => {
+      let a = document.createElement('a');
+      a.textContent = '...';
+      a.href = '#';
+
+      // attach event listener.
+      a.addEventListener('click', expand);
+
+      // Add it to the DOM
+      self.expandable = document.createElement('li');
```

Comment:
Why not just add it to a private property from the parent function?

Reply:
No idea... Updated it 😅